package/MDAnalysis/analysis/pca.py

```diff
@@ -246,8 +283,7 @@ def _prepare(self):
                 self.mean += self._atoms.positions.ravel()
             self.mean /= self.n_frames
 
-        self.mean_atoms = self._atoms
-        self.mean_atoms.positions = self._atoms.positions
```

Comment:
I realise it's not being used anywhere, but this might actually be worth keeping (if only commented off), I think this is part of an unintentionally killed code path that now leads to #2728

Reply:
Honestly, I would like to remove or only keep the positions. Looking forward to mda_cli I see no way to save a `AtomGroup` in way tu use it later without launching a python interpreter...

Reply:
There might simply be results that mdacli cannot save out of the box but they are necessary for programmatic downstream analysis. That's just something mdacli needs to deal with separately.